pep-0577.rst

```diff
@@ -302,46 +342,61 @@ the global scope instead of raising ``SyntaxError`` as it does today::
     >>> x
     1
 
+Note: this is just one potential implementation strategy for correctly
+handling binding of global targets in scoped expressions, and the development
+and documentation of a reference implementation may show that it's actually
+just as easy to propagate the explicit global variable declaration down into
+the nested scope as needed, which would avoid the need to make this change to
+the interaction between explicit ``nonlocal`` and ``global`` declarations.
```

Comment:
I strongly recommend doing that instead of allowing clashing explicit global and nonlocal declarations.